api/provider/aws/test_aws/service_create_test.go

```diff
@@ -104,6 +110,7 @@ func TestServiceCreate(t *testing.T) {
 	createServiceInput := &ecs.CreateServiceInput{}
 	createServiceInput.SetCluster("l0-test-env_id")
 	createServiceInput.SetDesiredCount(1)
+	createServiceInput.SetLaunchType(ecs.LaunchTypeFargate)
```

Comment:
I believe we also need to set the networking mode to `awsvpc`, which is the only networking mode that works w Fargate.

Reply:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_AWSCLI_Fargate.html as a point of reference

Reply:
I can't find a `SetNetworkingMode` method anywhere (it looks like `networkMode` is a field in a task definition file), but there _does_ exist `SetNetworkConfiguration`, which is talked about being set for tasks/services that _use_ the `awsvpc` networking mode. From what I can tell, a Fargate instance automatically/by-default uses the `awsvpc` networking mode. I haven't yet figured out what (if anything) needs to be set for the network configuration, though.

In any event, it looks like I have some more research to do. :+1:

Reply:
One way to test is to just create a sample fargate service / task and see if ECS renders it with the `awsvpc` networkMode option in the resulting task definition in the ECS UI. Maybe it doesn't need to be defined here explicitly?

more details on `networkMode`: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#network_mode

more details on `awsvpc` mode, specifically:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-networking.html

Reply:
✅